### PR TITLE
Cloud.drone.io

### PR DESCRIPTION
**Pull request by mmlb**
- [ ] Changelog updated

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,46 +1,66 @@
...
-pipeline:
-  lint:
-    group: ci
-    image: golangci/golangci-lint:v1.24.0
+steps:
+  - name: lint
+    image: golangci/golangci-lint:v1.23.8
     commands:
-      - golangci-lint run -v -D errcheck
+      - golangci-lint run -v --skip-dirs cmd/packetflow
```
Comment (mikemrm) [inline .drone.yml:10]: cmd/cacherc?

File: .drone.yml [hunk 2]
```diff
@@ -1,46 +1,66 @@
...
-    group: publish
+  - name: publish
     image: plugins/docker
     registry: quay.io
-    repo: quay.io/packet/cacher
-    tags:
-      - ${DRONE_BRANCH/\//-}
-      - ${DRONE_BUILD_NUMBER}-${DRONE_COMMIT_SHA}
+    settings:
+      repo: quay.io/packet/hegel
```
Comment (mikemrm) [inline .drone.yml:34]: Should this be the cacher repo in quay?

File: .drone.yml [hunk 3]
```diff
@@ -1,46 +1,66 @@
...
+      - build
+      - fmt
+      - lint
+      - test
+
+  - name: publish_tag
+    image: plugins/docker
+    registry: quay.io
+    settings:
+      repo: quay.io/packet/hegel
```
Comment (mikemrm) [inline .drone.yml:54]: Same, re: cacher repo

File: .drone.yml (hunk 1 above)
Comment (mmlb) [reply .drone.yml]: fixed, I copied from hegel but failed to `git checkout -p` properly

File: .drone.yml (hunk 2 above)
Comment (mmlb) [reply .drone.yml]: ditto

File: .drone.yml (hunk 3 above)
Comment (mmlb) [reply .drone.yml]: ditto

**Pull request merged**